### DISCUSSION TITLE
Make table headers clickable

### DISCUSSION THREAD

**Discussion by kiwichick888** [qbittorrent/search-plugins#201] [General]
Sorry, this has nothing to do with the plugins themselves but it would be great if the plugins tables on the unofficial search plugins page had clickable headers eg: so it's easier to see which plugins have the most recent updates.

**Comment by Scare1** [comment]
They look clickable/sortable to me. qBittorrent 4.4.5 64-bit

**Reply by kiwichick888** [reply]
Sorry for the confusion, but I'm talking about the Plugins page. In fact, it's actually the unofficial plugins page so I've updated my post to make it more clear.
